### PR TITLE
Full list of command line options in command line doc

### PR DESCRIPTION
**Pull request by pouleyKetchoupp**
Based on the descriptions from `--help` command.

### CODE REVIEW

**Comment by pouleyKetchoupp** [comment]
Glad to hear I wasn't the only one who wanted this :D

I've just added the note about debug options that you suggested. Let me know if it's accurate enough.

File: getting_started/editor/command_line_tutorial.rst [hunk 1]
```diff
@@ -11,6 +11,129 @@ entirely from the command line. Given the engine relies on almost no
...
+| ``--position <X>,<Y>``      | Request window position.                                                   |
++-----------------------------+----------------------------------------------------------------------------+
+| ``--low-dpi``               | Force low-DPI mode (macOS and Windows only).                               |
++-----------------------------+----------------------------------------------------------------------------+
+| ``--no-window``             | Disable window creation (Windows only). Useful together with ``--script``. |
++-----------------------------+----------------------------------------------------------------------------+
+
+**Debug options**
+
+**Note:** Debug options are only available in the editor and debug export templates (they require ``debug`` or ``release_debug`` build targets, see :ref:`doc_introduction_to_the_buildsystem_target` for more details).
```
Comment (Calinou) [inline getting_started/editor/command_line_tutorial.rst]: This should be refactored to be a Sphinx note block:

```suggestion
.. note::

    Debug options are only available in the editor and debug export templates
    (they require ``debug`` or ``release_debug`` build targets, see
    :ref:`doc_introduction_to_the_buildsystem_target` for more details).
```

**Comment by pouleyKetchoupp** [comment]
Thanks for the feedback! I've changed it to a proper note.

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**